### PR TITLE
 Added Prefab EndPoint

### PR DESCRIPTION
**Pull request by Kjusok**
Added Prefab EndPoint.
When crossing with "Player", an event occurs - LevelPassed.

### CODE REVIEW

File: Assets/Scripts/Gameplay/EndPoint.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using UnityEngine;
+
+namespace Gameplay
+{
+    public class EndPoint : MonoBehaviour
+    {
+        public event Action LevelPassed;
```
Comment (ega-ega) [inline Assets/Scripts/Gameplay/EndPoint.cs]: Remove This event

File: Assets/Scripts/Gameplay/EndPoint.cs [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        
+        
+        private void OnTriggerEnter(Collider other)
+        {
+            if (other.GetComponent<Player>())
+            {
+                LevelPassed?.Invoke();
```
Comment (ega-ega) [inline Assets/Scripts/Gameplay/EndPoint.cs]: Need Inject IGameRun to this class. And invoke finish

**Review by ega-ega — CHANGES_REQUESTED** [review]

File: Assets/Scripts/Gameplay/EndPoint.cs (hunk 2 above)
Comment (Kjusok) [reply Assets/Scripts/Gameplay/EndPoint.cs]: Done

File: Assets/Scripts/Gameplay/EndPoint.cs (hunk 1 above)
Comment (Kjusok) [reply Assets/Scripts/Gameplay/EndPoint.cs]: Done

**Pull request closed without merging**